File: src/main/java/info/opensigma/module/data/ModuleCategory.java
package info.opensigma.module.data;

public enum ModuleCategory {
    RENDER("Render"),
    PLAYER("Player"),
    COMBAT("Combat"),
    WORLD("World"),
    MISC("Misc"),
    EXPLOIT("Exploit"),
    MOVEMENT("Movement"),
    GUI("GUI"),
    ITEM("Item");

    public final String name;

    ModuleCategory(String name) {
        this.name = name;
    }
}

File: src/main/kotlin/info/opensigma/module/data/ModuleCategory.kt
package info.opensigma.module.data;

public enum ModuleCategory {
    GUI("Gui"),
    COMBAT("Combat"),
    RENDER("Render"),
    WORLD("World"),
    MISC("Misc"),
    PLAYER("Player"),
    ITEM("Item"),
    MOVEMENT("Movement");

    public final String name;

    ModuleCategory(String name) {
        this.name = name;
    }
}
